classes/options-general.php

```diff
@@ -901,11 +901,11 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 
 									<div class="postbox">
 
-										<h3><?php _e( 'Delete SimpleMap Data', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Delete SimpleMap Data', 'SimpleMap' ); ?></h3>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->